assets/model_monitoring/components/src/data_quality_compute_metrics/compute_data_quality_metrics.py

```diff
@@ -542,7 +551,7 @@ def compute_data_quality_metrics(df, data_stats_table):
         .when(col("dataType") == "DateType()", "Categorical")
         .when(col("dataType") == "FloatType()", "Numerical")
         .when(col("dataType") == "ShortType()", "Numerical")
-        .when(col("dataType") == "ByteType()", "Categorical")
+        .when(col("dataType") == "ByteType()", "Numerical")
```

Comment:
why making this change? When I checked with Alex he mentioned that Binary should be Categorical

Reply:
This is not Binary, this is byte type, when I recheck byte type in spark, it is numerical.